polyfills/Element/prototype/dataset/detect.js

```diff
@@ -0,0 +1,6 @@
+(document.documentElement.dataset &&
+	(
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset')  &&
+		Object.getOwnPropertyDescriptor(Element.prototype, 'dataset').get
```

Comment:
This is quite a new API to use for a detect.  Is it possible to detect without using ES5?